### ISSUE TITLE
Usage clarification

### ISSUE THREAD

**Issue by Jarli01** [vatesfr/xapi-explore-sr#5]
```
> xapi-explore-sr
Usage: xapi-explore-sr [--full] <SR UUID> <XenServer IP> <XenServer user> [<XenServer password>]
```

IE:

```
> xapi-explore-sr --full *********-****-****-****-************ 192.168.1.X root [enter]
 Xen password: *********
```

Usage clarifications.

**Comment by julien-f** [comment]
You mean *XenServer* instead of *Xen*?

**Comment by julien-f** [comment]
Closed with v0.0.4.
